pocket_lab/pl_gui_views.c

```diff
@@ -879,12 +885,33 @@ static void pl_gui_dmm_btn_event_cb(lv_event_t *event)
 		} else { // Start/Stop label
 			if (pl_gui_dmm_is_running) {
 				lv_label_set_text(label, "Start");
+
+				for (cnt = 0; cnt < pl_gui_dmm_chn_cnt; cnt++) {
+					if (lv_obj_get_state(pl_gui_dmm_chn_checkbox[cnt]) == LV_STATE_CHECKED)
+						lv_obj_clear_state(pl_gui_dmm_chn_checkbox[cnt],
+								   LV_STATE_CHECKED | LV_STATE_DISABLED);
+					else
```

Comment:
I suppose if-else blocks with single liner statements within also need to have brackets around. The rest of the file follows such a structure. Can we add brackets everywhere?